R/deduplicate.R

```diff
@@ -0,0 +1,206 @@
+##' Deduplicate an orderly archive.  Deduplicating an orderly archive
+##' will replace all files that have the same content with "hard
+##' links".  This is reliable only on Unix-like systems.  If you use
+##' this feature, it is \emph{very important} that you treat your
```

Comment:
Can you clarify "reliable only on Unix-like systems" ... does this mean we shouldn't do it, or can't do it, or.... 

I'm not super-familiar with this, but Windows has apparently supported hard links (of some sort) since Win 7 (https://www.howtogeek.com/howto/16226/complete-guide-to-symbolic-links-symlinks-on-windows-or-linux/) - so I would have thought Windows solution should be doable at some stage.... if we wanted to.

Reply:
means currently disabled, not sure how well it would work (see other comments for details)